modules/algebrasAndModel/src/main/scala/utils/Utils.scala

```diff
@@ -0,0 +1,45 @@
+package utils
+
+object Utils {
```

Comment:
❓ Maybe we could name this object like `Types`?

Reply:
Yes, I think it will be a great name.
Thank you 😊